### ISSUE TITLE
Connection pool exhausted

### ISSUE THREAD

**Issue by billybg** [kimios/kimios#20]
I have installed the latest release (1.2.1) with PostgreSQL but, leaving it running for a while (even though no users are logged in), I get the exception 
"ohejdbc.spi.SqlExceptionHelper - Connection pool exhausted - try increasing 'maxPoolSize' and / or 'borrowConnectionTimeout' on the DataSourceBean".

Maybe it is related to a bug in the version 3.9.3 of TransactionEssential library. The bug was fixed in 3.9.4 of ExtremeTransaction (only commercial license).

Is there a workaround to avoid the upgrade to the commercial version?

[kimios-stdout.log](https://github.com/kimios/kimios/files/3680681/kimios-stdout.log)

**Comment by devtom30** [comment]
Hello,
Thank you for your message. We will look at it very soon.
Have you an idea of the time needed to have the bug ? 
Regards,
Thomas

**Comment by billybg** [comment]
Hi,
I don't remember how much time is needed before the problem occurs, but I solved it modifying the retrieveDatabaseInfo() method in org.kimios.telemetry.controller.TelemetryController and adding the close connection command.

```
    public HashMap<String, String> retrieveDatabaseInfo() throws Exception {
    	Connection conn = this.databaseConnection.getConnection();
        DatabaseMetaData metaData = conn.getMetaData();

        HashMap<String, String> data = new HashMap<>();
        data.put("databaseProductName", metaData.getDatabaseProductName());
        data.put("databaseProductVersion", metaData.getDatabaseProductVersion());
        
        conn.close();
        return data;
    }
```

**Comment by devtom30** [comment]
Ok. Thank you.
Actually we work also on an other repository. This had been already fixed. We will synchronize this repository too.
Have a nice day

**Comment by veenarm** [comment]
@devtom30 - Have you synched the repositories anytime recently? looks like no actual code changes in this repo for 2years.

**Comment by devtom30** [comment]
Fix in 2053d2b44c22271567ed46b486441030aeff4572
